Source/source/GPS.cpp

```diff
@@ -51,6 +57,47 @@ string get_message() {
 }
 
 bool send_message(gps_data decoded_data) {
```

Comment:
If the message-passing objects from OS don't allow true/false return values on success/failure, this might as well be a void as well.

Reply:
That's a good point. I'll do some more digging into some of the methods to check if we can get a `bool` response from any of them.

Reply:
No functionality right now, dropped a [PR](https://github.com/AndrewWay/cubesat/pull/15) into the OS repo adding a bool response on success. Once that gets approved, I'll update this method with proper returns.